docs/dev-proc/source-code-structure.md

```diff
@@ -47,32 +39,71 @@ Dasharo Release tags in git repository use format: `<platform>_vX.Y.Z`
 
 ## New platform support
 
-Branch for new platform should be created from most recent`master` branch tag.
-If there is justified need to create support for new board at arbitrary
-non-tagged commit developer should mark this commit with `<platform>_v0.0.0`
-tag.
+Support for new platforms is submitted via PRs to the `dasharo` branch. All
+mainboards supported by Dasharo live in this branch.
 
 ## Force-pushes rules
 
-Force-pushes to `<platform>/rel_vX.Y.Z`, `<platform>/develop` or
-`<platform>/<feature>` are forbidden with following exceptions:
+Force-pushes to `dasharo` and `master` / `main` are forbidden with the
+following exceptions:
 
-* rebasing - when some other PR is merged to target branch before our does, or
-  when upstream's master introduces the same fixes that our branch would
-* squashing - to not produce unnecessary "fix indentation" or "add missing
-  braces" commits to the history
+* rebasing the `dasharo` branch on new coreboot versions
 * (re-)signing commits (both -S and -s) - shouldn't happen, but if it does
```

Comment:
Resigning should not be needed on dasharo branch. Unsigned commits should never land there via PRs because CI checks for it. Unsigned commit may land from upstream, but we won't sign those.